src/util/ParseableDuration.h

```diff
@@ -21,6 +21,20 @@ template <typename DurationType>
 class ParseableDuration {
   DurationType duration_{};
 
+ public:
+  // Implicit conversion is on purpose!
+  ParseableDuration() = default;
+  ParseableDuration(DurationType duration) : duration_{duration} {}
+  operator DurationType() const { return duration_; }
```

Comment:
The comment is one line too high, and you can explicitly mark the two implicit conversions as
`explicit(false)`. Sonarcloud will still complain (which is okay), but you have made the intent clear directly in the code.